lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -145,7 +145,6 @@ EOS
 # Please put the most recent first.
 my $sw_legacy_java = Genome::Config::get('sw_legacy_java');
 my @PICARD_VERSIONS = (
-    '1.123' => '/gscmnt/sata132/techd/solexa/jwalker/lib/picard-tools-1.123',
```

Comment:
Don't we need version 1.123 here ? just thought it should be $sw_legacy_java . '/samtools/picard-tools-1.123'